app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarComponent.kt

```diff
@@ -26,6 +26,7 @@ class ToolbarComponent(
     bus: ActionBusFactory,
     private val sessionId: String?,
     private val isPrivate: Boolean,
+    private val inSearchFragment: Boolean,
```

Comment:
Could we rename this to "startInEditMode"?

Reply:
yas